docs/conf.py

```diff
@@ -54,15 +54,8 @@
 bibtex_footbibliography_header = bibtex_bibliography_header
 
 # html theme options
-html_theme = 'qiskit_sphinx_theme'
-html_last_updated_fmt = '%Y/%m/%d'
```

Comment:
Oh wait, this should still be there.

Reply:
You mean the `html_last_updated_fmt` should remain?

Reply:
(the updated `html_theme` is being displayed at the bottom of this change block)